Source/Core/DolphinQt2/Debugger/MemoryWidget.cpp

```diff
@@ -0,0 +1,575 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Debugger/MemoryWidget.h"
+
```

Comment:
`<string>` and `<vector>` should be included.

Reply:
Done.